tempesta_fw/t/functional/helpers/chains.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import print_function
+import copy
+from helpers import tf_cfg, deproxy, tempesta
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+def base(uri='/', method='GET', forward=True, date=None):
+    """Base message chain. Looks like simple Curl request to Tempesta and
+    response for it.
+
+    Return new message chain.
+    """
+
+    if date is None:
+        date = deproxy.HttpMessage.date_time_string()
+
+
+    #
+    # Build a request
+    #
+
+    request_headers = [
+        'Host: %s' % tf_cfg.cfg.get('Tempesta', 'ip'),
+        'User-Agent: curl/7.53.1',
+        'Connection: keep-alive',
+        'Accept: */*'
+    ]
+    request = deproxy.Request.create(method, request_headers, uri=uri)
```

Comment:
I don't quite understand what you want me to do — the GET chain, as basic as it gets, is still valid for every method.